packages/migration/src/migrations/user-mgnt/20240628124653-remove-userroles.ts

```diff
@@ -17,7 +17,10 @@ export const up = async (db: Db, client: MongoClient) => {
     await session.withTransaction(async () => {
```

Comment:
I think we need to create a new migration for these changes as some countries could have already ran this migration in their environments